routemodel/data_retrieval/elevations.py

```diff
@@ -89,38 +87,36 @@ def get_elevation_data(coordinates_str: str, method='default', sample_val=0, hei
     """
     if method == 'default':
         # append specified elevation URL details
-        url = BASE_URL + 'Elevation/List?'
+        query = 'Elevation/List?'
         # add parameters to URL
-        url += 'points={}&heights={}&key={}'.format(coordinates_str, heights, API_KEY)
+        query += 'points={}&heights={}'.format(coordinates_str, 
+                                               heights)
     elif method == 'polyline':
         if sample_val <= 0:
             print("Error, sample_val must be greater than 0")
             sys.exit()
         # append specified elevation URL details
-        url = BASE_URL + 'Elevation/Polyline?'
+        query = 'Elevation/Polyline?'
         # add parameters to URL
-        url += 'points={}&heights={}&samples={}&key={}'.format(coordinates_str, heights, sample_val, API_KEY)
+        query += 'points={}&heights={}&samples={}'.format(coordinates_str, 
+                                                          heights, 
+                                                          sample_val)
     else:
         print("Error, incorrect method parameter")
         sys.exit()
 
     # request URL and return json-encoded content of a response
-    try:
-        response = requests.get(url)
-        response.raise_for_status()
-    except requests.exceptions.RequestException as err:
-        print("An error occurred:", err, "\nMessage:", err.response.text)
-        sys.exit()
+    return query
 
-    return response.json()
 
-
-def parse_elevation_data(response: dict, coordinates: list, method='default'):
+def parse_elevations_data(response: dict, coordinates: list = [], method='default'):
     """
     Parsing through Elevations API call response.
         - if API call was done using polyline method, set method in parse_elevation_data to 'polyline'
```

Comment:
Nitpicky, but can we say: (add s to method name, make sure it specifies that method is the parameter name)

```suggestion
..., set method parameter in `parse_elevations_data` to 'polyline'
```